test/gui/config.sample.ini

```diff
@@ -1,6 +1,7 @@
 [DEFAULT]
-CLIENT_SYNC_PATH=
-BACKEND_HOST=
-CLIENT_SYNC_TIMEOUT=
-MIDDLEWARE_URL=
-CLIENT_LOG_FILE=
+CLIENT_SYNC_PATH_USER1=/tmp/client-bdd-user1/
```

Comment:
no need to set this values in the sample ini file, those are anyway the defaults
the ini file should also work if no setting is set, because they are taken from `gui/shared/scripts/bdd_hooks.py `